server/spec/models/user_spec.rb

```diff
@@ -7,7 +7,8 @@
     described_class.new(
       nick_name: "tom",
       email: "t@example.com",
-      password: "Qwerty123"
+      password: "Qwerty123",
+      password_confirmation: "Qwerty123"
     )
   end
```

Comment:
Would be good to use gem `factory_bot_rails` for defining such test data
ref: https://github.com/thoughtbot/factory_bot_rails